options.js

```diff
@@ -1,13 +1,33 @@
 
+const file_input = document.getElementById("file-input")
 const file_selector = document.getElementById("file-selector")
 
 class Options {
+
+  constructor() {
+    this.backButton = createButton('back')
```

Comment:
Use the new painted buttons... let's start consolidating the UI by getting rid of these html buttons.